packages/terra-button-group/src/terra-dev-site/doc/button-group/ButtonGroup.1.doc.mdx

```diff
@@ -1,3 +1,16 @@
+import { Badge } from 'terra-button-group/package.json?dev-site-package';
+
+import ButtonGroupWithIcons from '../example/ButtonGroupWithIcons?dev-site-example';
+import ButtonGroupDisabledButtons from '../example/ButtonGroupDisabledButtons?dev-site-example';
+import ButtonGroupSingleSelect from '../example/ButtonGroupSingleSelect?dev-site-example';
+import ButtonGroupMultiSelect from '../example/ButtonGroupMultiSelect?dev-site-example';
+import ButtonGroupIsBlock from '../example/ButtonGroupIsBlock?dev-site-example';
+
+import ButtonGroupPropsTable from '../../../../src/ButtonGroup?dev-site-props-table';
```

Comment:
Apply this everywhere you're traversing the folder structure so much.
```suggestion
import ButtonGroupPropsTable from 'terra-button-group/src/ButtonGroup?dev-site-props-table';
```

Reply:
updated here https://github.com/cerner/terra-core/pull/2815/commits/994a492f7ec5a562f67aa3285a2b49bf117cef0e.